aries_vcx_core/Cargo.toml

```diff
@@ -5,36 +5,34 @@ edition = "2021"
 
 [features]
 ########################## DEP FLAGS ################################
-# Feature flag to include the libvdrtools dependency
-vdrtools_anoncreds = ["dep:libvdrtools", "dep:indy-api-types"]
 vdrtools_wallet = ["dep:libvdrtools", "dep:indy-api-types"]
 # Feature flag to include the 'modular library' dependencies (vdrtools alternatives; indy-vdr, indy-credx)
-modular_libs = ["dep:indy-credx"]
-vdr_proxy_ledger = ["modular_libs", "dep:indy-vdr-proxy-client"]
+credx = ["dep:indy-credx"]
+vdr_proxy_ledger = ["credx", "dep:indy-vdr-proxy-client"]
```

Comment:
general question that doesn't need to be answered here: does `vdr_proxy_ledger` (i.e. where `VdrProxySubmitter` is the core component) **actually** need the `credx` feature?

You guys know this part of the code best, but it seems like the only reason `vdr_proxy_ledger` needs `credx` is because `vdr_proxy_ledger` is being used to hide the `VdrProxyProfile` implementation, which depends on `IndyCredxAnonCreds` (and `IndySdkWallet` too actually (`vdrtools_wallet`?)). I wonder if logically `VdrProxyProfile` should be hidden by `credx` AND `vdr_proxy_ledger` rather than making `vdr_proxy_ledger` always depend on `credx`

Reply:
I guess it's a matter of perspective about whether the primary "feature" of `vdr_proxy_ledger` is the `VdrProxySubmitter` AND `VdrProxyProfile`, or if it's just the `VdrProxySubmitter`

Reply:
Idk. @mirgee worked on this. Any insights? I just replaced the underlying implementation being used in the `VdrProxyProfile` since we're deprecating `libvdrtools`.

Reply:
ye no worries, just a passing comment